pages/platform/kubernetes-k8s/resizing-persistent-volumes/guide.en-gb.md

```diff
@@ -49,15 +49,14 @@ You also need to know how PVs are handled on OVHcloud Managed Kubernetes service
 
```

Comment:
Could you add on top of the page the following information to inform the reader of the latest documentation update?

```text
**Last updated January 06<sup>th</sup>, 2020.**
```